config.rb

```diff
@@ -76,6 +76,8 @@ def self.test?
   override :versioning, false, bool
   optional :hetzner_user, string, clear: true
   optional :hetzner_password, string, clear: true
+  override :hetzner_hostname, string
+  override :hetzner_server_id, string
```

Comment:
This seems like too generic a name for a server that gets wiped all the time. And although it terminologically lines up with the Hetzner API, in the somewhat high level context of Config, its purpose is mysterious. I kinda liked your old ci_hetzner_server_id, although I wish there was a an even better name to signal that the program will do something destructive to that server.